lang/cs/Org.Apache.REEF.Common/Runtime/Evaluator/EvaluatorSettings.cs

```diff
@@ -174,12 +187,46 @@ public IRemoteManager<REEFMessage> RemoteManager
             }
         }
 
-        public IInjector Injector
+        public IInjector ClrDriverConfigInjector
         {
             get
             {
                 return _injector;
             }
         }
+
+        private Optional<TaskConfiguration> GetTaskConfiguration()
```

Comment:
You mean a property? It is really a method. I renamed it to avoid using Get. 
I was thinking to put RootServiceConfiguration as a named parameter in the constructor so that we don't need to GetNamedInstance here. But it is optional. Putting it in the constructor would cause injection exception if no value is provided in the configuration. In this method, I can use catch to skip the injection exception if the value is not given.